synthetics/test_conversions.go

```diff
@@ -221,7 +220,7 @@ func resourceDataToTest(d *schema.ResourceData) (*synthetics.V202101beta1Test, e
 	test := synthetics.NewV202101beta1Test()
 	test.SetName(d.Get("name").(string))
 	test.SetType(d.Get("type").(string))
-	test.SetDeviceId(d.Get("device_id").(string))
+	test.SetDeviceId("0")
```

Comment:
Please put a comment why Device ID is hardcoded to 0 and not taken from the user

Reply:
Addressed: c564f604a57155de603151b7e17f528bfdcb00d8